include/libtorrent/buffer.hpp

```diff
@@ -68,7 +68,10 @@ class buffer
 
 		if (size == 0) return;
 
-		size = (size + 7) & (~std::size_t(0x7)); //round_up8(size);
+		// this rounds up the size to be 8 bytes aligned
+		// it mostly makes sense for platforms with support
```

Comment:
heh, maybe I miswrote. I meant to say "without". the ``malloc_size()`` rounds up to the actual slab size